udp-ballerina/Ballerina.toml

```diff
@@ -1,27 +1,27 @@
 [package]
 org = "ballerina"
 name = "udp"
-version = "@toml.version@"
+version = "0.8.0"
```

Comment:
Seems this is mistakenly committed. Refer the other such changes in `Ballerina.toml` file as well.

Reply:
File reverted